typescript/sdk/src/core/TestRecipientDeployer.ts

```diff
@@ -45,15 +44,21 @@ export class TestRecipientDeployer extends HyperlaneDeployer<
     chain: ChainName,
     config: TestRecipientConfig,
   ): Promise<TestRecipientContracts> {
+    this.logger(`Deploying TestRecipient on ${chain}`, config);
     const testRecipient = await this.deployContract(chain, 'testRecipient', []);
```

Comment:
pretty sure the deployContract and configureIsm already kinda log something like this